src/main/java/org/kiwiproject/eureka/EmbeddedEurekaServer.java

```diff
@@ -59,7 +57,7 @@ public EmbeddedEurekaServer(String basePath) {
 
         var webContext = new WebAppContext();
         webContext.setContextPath(basePath);
-        webContext.setResourceBase(pathFromResourceName("webapp").toString());
+        webContext.setResourceBase(Resources.getResource("webapp").toString());
```

Comment:
Curious why the kiwi `pathFromResourceName` can't be used anymore?

Reply:
`pathFromResourceName` worked in the test cause it was on the filesystem, it did not work when it was inside the jar (i.e. testing from service-discovery-client)